valhalla/sif/dynamiccost.h

```diff
@@ -679,6 +685,12 @@ class DynamicCost {
    */
   virtual void SetAllowTransitConnections(const bool allow);
 
+  /**
+   * Sets the flag indicating whether edges with valid restriction conditional=destination are
+   * allowed.
+   */
+  void set_allow_conditional_destination_restrictions(const bool allow);
```

Comment:
After some contemplation I came to the conclusion that it's useless. In this PR we handle `conditional=destination` as an ordinary time restriction. As far as we do not relax any restrictions in case of first run of path algorithm failed, I don't see any reason why we should do it for edges with `conditional=destination` tag. We will simplify code a lot if remove this parameter
@maksimandrianov @SvetlanaBayarovich WDYT ?

Reply:
ok, looks like we should route through such roads in case of there are no other ways.